pkg/destroy/gcp/gcp.go

```diff
@@ -176,6 +195,29 @@ func (o *ClusterUninstaller) getZoneName(zoneURL string) string {
 	return ""
 }
 
+// getRegionFromZone extracts a region name from a zone name of the form: us-central1-a
+// Splitting the name with the last delimiter `-`, leaves a string like: us-central1
+func (o *ClusterUninstaller) getRegionFromZone(zoneName string) string {
```

Comment:
Surely there's a utility for this already ... ?

Reply:
The zone has a field for region, but this requires an API call to obtain. Might be better to do it this way. I don't know that we have this anywhere else in the code.